utils/steward-cli/src/commands/info/view_state.rs

```diff
@@ -1,71 +1,118 @@
-use std::sync::Arc;
-
+use anchor_lang::AccountDeserialize;
 use anyhow::Result;
-use jito_steward::StewardStateAccount;
+use jito_steward::{utils::ValidatorList, Config, StewardStateAccount};
 use solana_client::nonblocking::rpc_client::RpcClient;
-use solana_sdk::pubkey::Pubkey;
-use spl_stake_pool::{find_stake_program_address, find_transient_stake_program_address};
+use solana_sdk::{account::Account, pubkey::Pubkey};
+use spl_stake_pool::{
+    find_stake_program_address, find_transient_stake_program_address, state::StakeStatus,
+};
+use std::{collections::HashMap, sync::Arc};
+use validator_history::ValidatorHistory;
+
+use crate::commands::command_args::ViewState;
 
-use crate::{
-    commands::command_args::ViewState,
-    utils::accounts::{get_all_steward_accounts, UsefulStewardAccounts},
+use stakenet_sdk::utils::{
+    accounts::{get_all_steward_accounts, get_validator_history_address},
+    debug::{format_simple_steward_state_string, format_steward_state_string},
 };
 
 pub async fn command_view_state(
     args: ViewState,
     client: &Arc<RpcClient>,
     program_id: Pubkey,
 ) -> Result<()> {
+    println!("Fetching a lot of accounts, please use a custom RPC for better performance");
+
     let steward_config = args.view_parameters.steward_config;
 
-    let steward_state_accounts =
+    let all_steward_accounts =
         get_all_steward_accounts(client, &program_id, &steward_config).await?;
 
     if args.verbose {
-        _print_verbose_state(&steward_state_accounts);
+        let vote_accounts: Vec<Pubkey> = all_steward_accounts
+            .validator_list_account
+            .validators
+            .iter()
+            .map(|validator| validator.vote_account_address)
+            .collect();
+
+        let history_accounts_to_fetch: Vec<Pubkey> = vote_accounts
+            .iter()
+            .map(|vote_account| {
+                get_validator_history_address(vote_account, &validator_history::id())
+            })
+            .collect();
+
+        // let raw_history_accounts = client
+        //     .get_multiple_accounts(&history_accounts_to_fetch)
+        //     .await?;
+
+        let raw_history_accounts: Vec<Option<Account>> =
+            vec![None; history_accounts_to_fetch.len()];
```

Comment:
i think we added this to avoid the 413 Payload too big or segfault or something, were these history accounts important?

Reply:
They are only important to see if something is blacklisted or not

Reply:
Ah right. Not super critical

Reply:
Done